lib/puppet/provider/cs_location/crm.rb

```diff
@@ -16,9 +16,21 @@
   # Path to the crm binary for interacting with the cluster configuration.
   # Decided to just go with relative.
   commands crm: 'crm'
+  
+  # Path to the pacemakerd binary to check version for resource_discovery feature
+  # Decided to just go with relative.
+  commands :pacemakerd => 'pacemakerd'
 
   mk_resource_methods
 
+  # we need to check if we run at least pacemakerd version 1.1.13 before enabling feature discovery
+  # see http://blog.clusterlabs.org/blog/2014/feature-spotlight-controllable-resource-discovery
+  pacemakerd_version_string = pacemakerd('--version')
+  pacemakerd_version = pacemakerd_version_string.scan(%r{\d+\.\d+\.\d+}).first unless pacemakerd_version_string.nil?
+  if Puppet::Util::Package.versioncmp(pacemakerd_version,'1.1.13') >= 0
```

Comment:
what happens if `pacemakerd_version` is nil?

Reply:
@bastelfreak Good point. In my newer pushes i tried to catch the issue. I also tried to catch the issue if the pacemakerd command is not available at all yet (f.e. on fresh installations).
I would like to throw errors in not yet existing else-parts of the ifs (for the case that the pacemakerd devs ever change the output of "pacemakerd --version"). I tried that, but i am too dumb to do so, can you please help me? (Like you described it in you great talk, which was just published on youtube recently 😃  ...by the way, we live just a few dozen kilometers away from each other.. 😃 )

Reply:
Uargh, i saw the rubocop issues. I will (try to) fix them, however i am still interested how i can throw proper errors as described before...